docs/configuration.html

```diff
@@ -267,23 +267,38 @@ <h3 class="anchor-heading"><a id="adminclientconfigs" class="anchor-link"></a><a
   Below is the configuration of the Kafka Admin client library.
   <!--#include virtual="generated/admin_client_config.html" -->
 
-  <h3 class="anchor-heading"><a id="systemproperties" class="anchor-link"></a><a href="#systemproperties">3.8 System Properties</a></h3>
+  <h3 class="anchor-heading"><a id="mirrormakerconfigs" class="anchor-link"></a><a href="#mirrormakerconfigs">3.8 MirrorMaker Configs</a></h3>
+  Below is the configuration of the connectors that make up MirrorMaker 2.
+  <h4 class="anchor-heading"><a id="mirrormakersourceconfigs" class="anchor-link"></a><a href="#mirrormakersourceconfigs">3.8.1 MirrorMaker Source Configs</a></h4>
+  Below is the configuration of MirrorMaker 2 source connector for replicating topics.
+  <!--#include virtual="generated/mirror_source_config.html" -->
+  <h4 class="anchor-heading"><a id="mirrormakercheckpointconfigs" class="anchor-link"></a><a href="#mirrormakercheckpointconfigs">3.8.2 MirrorMaker Checkpoint Configs</a></h4>
+  Below is the configuration of MirrorMaker 2 checkpoint connector for emitting consumer offset checkpoints.
+  <!--#include virtual="generated/mirror_checkpoint_config.html" -->
+  <h4 class="anchor-heading"><a id="mirrormakerheartbeatconfigs" class="anchor-link"></a><a href="#mirrormakerheartbeatconfigs">3.8.3 MirrorMaker HeartBeat Configs</a></h4>
+  Below is the configuration of MirrorMaker 2 heartbeat connector for checking connectivity between connectors and clusters.
+  <!--#include virtual="generated/mirror_heartbeat_config.html" -->
+  <h4 class="anchor-heading"><a id="mirrormakercommonconfigs" class="anchor-link"></a><a href="#mirrormakercommonconfigs">3.8.4 MirrorMaker Common Configs</a></h4>
+  Below is the common configuration properties that apply to all three connectors above.
```

Comment:
Nit: plural
```suggestion
  Below are the common configuration properties that apply to all three connectors above.
```

Also, IMO it makes more sense for the common properties to come first (someone may be expecting, e.g., `source.cluster.alias` to be included in the docs for a connector, and might be confused not to see it before scrolling further).